pandas/tests/frame/test_reshape.py

```diff
@@ -253,9 +253,11 @@ def test_unstack_fill(self):
         # By default missing values will be NaN
         result = data.unstack()
         expected = DataFrame({'a': pd.Categorical(list('axa'),
-                                                  categories=list('abc')),
```

Comment:
so we explicity force these to ordered? (or is it starting with an ordered cat)? this seems odd.